core/src/main/java/io/questdb/griffin/SqlCompiler.java

```diff
@@ -3125,6 +3114,29 @@ private CompiledQuery sqlTableBackup(SqlExecutionContext executionContext) throw
         }
     }
 
+    public class QueryBuilder implements Mutable {
```

Comment:
nit: I'd make it an interface implementation instead of exposing the inner class

Reply:
I did think of this but I feel this is too soon. We can introduce interface any time.